src/pages/Applications/List/ApplicationsList.view.tsx

```diff
@@ -21,6 +22,10 @@ const ApplicationsListView = ({ applications }: Props) => {
               from: application.salaryRange.from,
               to: application.salaryRange.to,
             }}
+            availableStatuses={values(ApplicationStatus)}
```

Comment:
I think there is no need to use lodash here as you can achieve the same using natvie JS methods Object.values

Reply:
I like lodash :D I don't like doing Object.something :D 
I can change it if @jakobos also wants to use pure JS.

Reply:
If we have lodash dependency, I think that it is no shame to use it